### PR TITLE
Move all data to common folder

### PR DESCRIPTION
**Pull request by sbalandi**

### CODE REVIEW

**Comment by eaidova** [comment]
@sbalandi general structure is good, thanks!
I have several suggestions regarding some files movements:
* data/text/vocab.txt -> data/text/bert-uncased/vocab.txt - I believe, in future we can have several files with the same name, so let's help to identify correctly model which this vocab corresponds.
* data/image/annotation and data/json/annotation -> data/datasets/person_detection (and place annotation and images in the same folder), it is parts of the same dataset, spliting them to different folders may confuse users

**Comment by sbalandi** [comment]
looks like 208-optical-character-recognition.ipynb will be fail anyway because of this link:
`image_file = "https://github.com/openvinotoolkit/openvino_notebooks/raw/main/notebooks/data/image/intel_rnb.jpg" `
This link is on main branch, but in branch of this commit folder structure changed.

**Comment by eaidova** [comment]
> image_file = "https://github.com/openvinotoolkit/openvino_notebooks/raw/main/notebooks/data/image/intel_rnb.jpg"

@sbalandi can we replace link with relative path?

File: notebooks/219-knowledge-graphs-conve/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 Commonly, knowledge graphs are used to encode domain information by representing them as factual triples of source, relation and target entities.
 For ex. For the countries domain, the domain knowledge can be encoded by representing the countries as entities (for ex. Germany, Poland) and relation between them (Neighbor) as factual triples (Poland, Neighbor, Germany) as well as other information as follows:
 <p style="text-align:center;">
-    <img src="./images/countries_knowledge_graph.png" width=500>
+    <img src="../data/image/countries_knowledge_graph.png" width=500>
```
Comment (adrianboguszewski) [inline notebooks/219-knowledge-graphs-conve/README.md:8]: An URL of the image uploaded to Github should replace this like it's done in other README files. The countries_knowledge_graph.png file should be removed.

**Review by adrianboguszewski — CHANGES_REQUESTED** [review]
Good job!
Please also change the name of the example.png file to something more meaningful (e.g., licence_plate)

**Review by adrianboguszewski — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

**Comment by adrianboguszewski** [comment]
There is still usage in utils, which makes CI fail.

**Pull request merged**